.changeset/shy-sites-write.md

```diff
@@ -0,0 +1,5 @@
+---
+"snapwp": minor
```

Comment:
Changesets is not semantic 

```suggestion
"snapwp": patch
```

Reply:
@Swanand01 you changed the wrong thing in https://github.com/rtCamp/snapwp/pull/140/commits/af667b71cfb046b862e805f8f3adafffd81cdc8f